config/services.yaml

```diff
@@ -5,6 +5,10 @@
 # https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
     locale: 'fr'
+    mailchimp.list_id: '%env(NL_LIST_ID)%'
+    mailchimp.api_url: 'https://%env(API_SERVER)%.api.mailchimp.com/3.0/lists/%env(NL_LIST_ID)%/members'
```

Comment:
`mailchimp.api_url: '%env(MAILCHIMP_API_URL)/lists/%env(NL_LIST_ID)%/members'`
With full root api url in `MAILCHIMP_API_URL`